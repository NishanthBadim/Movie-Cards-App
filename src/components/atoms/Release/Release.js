import React from 'react';

const Release =(props) => {
    return(
        <h6>{props.year}</h6> 
    );
}
export default Release;