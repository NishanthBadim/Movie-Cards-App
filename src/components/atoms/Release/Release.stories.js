import React from 'react'
import { withKnobs, number } from '@storybook/addon-knobs/react'
import Release from './Release'

export default {
    component: Release,
    title: "Release",
    decorators: [withKnobs]
}

export const Default = () => {
    return (
        <Release year={number("year", 2019)} />
    )
}