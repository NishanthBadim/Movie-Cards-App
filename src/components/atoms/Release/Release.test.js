import React from 'react';
import {create} from 'react-test-renderer'
import "@testing-library/jest-dom/extend-expect";
import Release from './Release'

test("Match the snapshot - Release",() => {
    const input=create(<Release />);
    expect(input.toJSON()).toMatchSnapshot();
})