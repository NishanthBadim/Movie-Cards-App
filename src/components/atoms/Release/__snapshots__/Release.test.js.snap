// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the snapshot - Release 1`] = `<h6 />`;
